source/concepts/aggregate-transaction.rst

```diff
@@ -6,23 +6,53 @@ Aggregate Transaction
 
 Aggregated Transactions merge multiple transactions into one, allowing **trustless swaps**, and other advanced logic. NEM does this by generating a one-time disposable smart contract. When all involved :doc:`accounts<../concepts/account>` have cosigned the aggregate transaction, all the inner transactions are executed at the same time.
 
-********
-Examples
-********
+.. _aggregate-complete:
+
+******************
+Aggregate complete
+******************
+
+An aggregate transaction is  **complete** when all cosigners (for multisignature account) have signed it.
+
+The cosigners can sign the transaction without using the blockchain (offline through their own localhost). Once it has all the required signatures, one of them can announce it to the network. If the inner transaction setup is valid, and there is no validation error, the transactions will get executed at the same time.
```

Comment:
``(offline through their own localhost)`` I would remove or rephrase (offline through their own localhost), since it can be online (using the internet).